src/Symfony/Component/Console/Application.php

```diff
@@ -365,9 +365,9 @@ public function isAutoExitEnabled()
      *
      * @param bool $boolean Whether to automatically exit after a command execution or not
      */
-    public function setAutoExit($boolean)
+    public function setAutoExit(bool $boolean)
```

Comment:
`@param` annotation can be removed. From this and other methods changed by this PR.

Reply:
done